src/main/java/edu/tamu/app/service/manager/GitHubService.java

```diff
@@ -0,0 +1,227 @@
+    }
+
+    private int getPrimaryWorkItemCount(final String type, final GHProject project, final List<GHLabel> labels)
+            throws IOException {
+        label = getLabelByName(labels, type);
+        return project.listColumns()
+            .asList()
+            .stream()
+            .map(this::countCardsOnColumn)
+            .reduce(0L, (a, b) -> a + b)
+            .intValue();
+    }
+
+    private GHLabel getLabelByName(final List<GHLabel> labels, final String name) {
+        return labels.stream()
+            .filter(label -> label.getName().equals(name))
+            .findFirst()
+            .get();
+    }
+
+    private boolean cardContainsLabel(GHProjectCard card) {
+        try {
+            return card.getContent().getLabels().contains(label);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private long countCardsOnColumn(GHProjectColumn column) {
+        try {
+            return column.listCards()
+                .asList()
+                .stream()
+                .filter(this::cardContainsLabel)
+                .count();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private RemoteProject buildRemoteProject(GHProject project, List<GHLabel> labels) throws IOException {
+        final String scopeId = String.valueOf(project.getId());
+        final String name = project.getName();
+        final int requestCount = getPrimaryWorkItemCount(REQUEST_LABEL, project, labels);
+        final int issueCount = getPrimaryWorkItemCount(ISSUE_LABEL, project, labels);
+        final int featureCount = getPrimaryWorkItemCount(FEATURE_LABEL, project, labels);
+        final int defectCount = getPrimaryWorkItemCount(DEFECT_LABEL, project, labels);
+        return new RemoteProject(scopeId, name, requestCount, issueCount, featureCount, defectCount);
+    }
+
+    private List<Card> getCards(GHProject project) throws IOException {
+        List<Card> cards = new ArrayList<Card>();
+        for (GHProjectColumn column : project.listColumns().asList()) {
+            List<GHProjectCard> projectCards = column.listCards().asList();
+            AtomicInteger cardNumber = new AtomicInteger();
+            for (GHProjectCard card : projectCards) {
+                GHIssue content = card.getContent();
+
+                String id = String.valueOf(card.getId());
+                String name = content.getTitle();
+                String number = String.valueOf(cardNumber.getAndIncrement());
+                // TODO: Figure out what the type priority is
+                String type = "";
+                String description = content.getBody();
+                String status = card.getColumn().getName();
+                // TODO: Figure out how we want to handle sizes
+                String estimate = null;
```

Comment:
I would like to propose estimates be labels in which their value can be parsed to a float. `Float.parseFloat(String value)`

Reply:
Maybe validate them by a set of point estimates we use.

Reply:
Yeah, simple check if the label value is contained with a static list of points.

Reply:
I've sent a write-up to @jcreel and @jmicah detailing the 3 areas where we need to establish a convention. I might schedule a meeting to hash this out next week.